lean/main/03_expressions.lean

```diff
@@ -1,28 +1,27 @@
 /-
 # Expressions
 
-Expressions (terms of type `Expr`) are an abstract syntax tree for Lean
-programs. This means that each term which can be written in Lean has a
-corresponding `Expr`. For example, the application `f e` is represented by the
-expression `Expr.app ⟦f⟧ ⟦e⟧`, where `⟦f⟧` is a representation of `f` and `⟦e⟧`
-a representation of `e`. Similarly, the term `Nat` is represented by the
-expression ``Expr.const `Nat []``. (The backtick and empty list are discussed
-below.)
-
-The ultimate purpose of a Lean tactic block is to generate a term which serves
-as a proof of the theorem we want to prove. Thus, the purpose of a tactic is to
-produce (part of) an `Expr` of the right type. Much metaprogramming therefore
-comes down to manipulating expressions: constructing new ones and taking apart
-existing ones.
-
-Once a tactic block is finished, the `Expr` is sent to the kernel, which checks
-whether it is well-typed and whether it really has the type claimed by the
-theorem. As a result, tactic bugs are not fatal: if you make a mistake, the
-kernel will ultimately catch it. However, many internal Lean functions also
-assume that expressions are well-typed, so you may crash Lean before the
-expression ever reaches the kernel. To avoid this, Lean provides many functions
-which help with the manipulation of expressions. This chapter and the next
-survey the most important ones.
+Expressions (terms of type `Expr`) are an abstract syntax tree for Lean programs.
+This means that each term which can be written in Lean has a corresponding `Expr`.
+For example, the application `f e` is represented by the expression `Expr.app ⟦f⟧ ⟦e⟧`,
+where `⟦f⟧` is a representation of `f` and `⟦e⟧` a representation of `e`.
+Similarly, the term `Nat` is represented by the expression ``Expr.const `Nat []``.
+(The backtick and empty list are discussed below.)
+
+The ultimate purpose of a Lean tactic block
```

Comment:
Splitting this line here is intentional?

I personally tend to use semantic linefeeds but essentially nearly *never* break on anything but sentence endings (.!?) even though yeah breaking on a , or ; is allowed/can be fine and essentially meets the "Reason" for using semantic linefeeds.

But breaking on spaces is a bit more iffy I think.